charts/flyte-core/templates/admin/flytescheduler.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.flytescheduler.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "flytescheduler.name" . }}
+  namespace: {{ template "flyte.namespace" . }}
+  labels: {{ include "flytescheduler.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.flytescheduler.replicaCount }}
```

Comment:
Can we disallow this to be configurable and always keep it 1 . We can make it configurable when we allow scaling scheduler .